src/api/VoteMonitor.Api.Form/Mappers/HierarchicalMapper.cs

```diff
@@ -0,0 +1,70 @@
+﻿using AutoMapper;
+using System.Collections.Generic;
+using System.Linq;
+using VoteMonitor.Entities;
+
+namespace VoteMonitor.Api.Form.Mappers
+{
+    /// <summary>
+    /// Base class using Template Design Pattern in order to facilitate the mapping of hierarchical objects.
```

Comment:
Probably this comment needs to be updated

Reply:
Updated, thanks